contrib/openstack/README.md

```diff
@@ -1,3 +1,14 @@
-# Provision a Deis Cluster on OpenStack
+# Provision a Deis Cluster on Openstack
 
 Please refer to the instructions at http://docs.deis.io/en/latest/installing_deis/openstack/.
+
+```
+$ make discovery-url
+$ terraform apply \
+      -var "username=$OS_USERNAME" \
+      -var "password=$OS_PASSWORD" \
+      -var "tenant=$OS_TENANT_NAME" \
+      -var "auth_url=$OS_AUTH_URL" \
+      -var "image=coreos-717.3.0" \
+      contrib/openstack
```

Comment:
I think you might be missing a few vars in here... this README.md should also ideally point to the official docs (installing Deis on OpenStack) once this PR gets finalized.